backend/organization/models/members.py

```diff
@@ -19,10 +19,53 @@ class ProjectMember(models.Model):
 
     role = models.ForeignKey(
         Role, related_name="project_role",
-        verbose_name="Role", help_text="Points to user role",
+        verbose_name="Role(permissions)", help_text="Points to user role",
         on_delete=models.PROTECT
     )
 
+    role_in_project = models.CharField(
+        help_text="Points to the role of the person in the project, e.g. 'project manager'",
+        verbose_name="Role in project",
+        max_length=1024,
+        null=True,
+        blank=True
+    )
+
+    # TODO: use availability table
+    NOT_SPECIFIED = "not_specified"
+    ONE_TO_TWO = "1-2"
+    THREE_TO_FIVE = "3-5"
+    SIX_TO_TEN = "6-10"
+    ELEVEN_TO_FIFTEEN = "11-15"
+    SIXTEEN_TO_TWENTY = "16-20"
+    TWENTYONE_TO_TWENTYFIVE = "21-25"
+    TWENTYSIX_TO_THIRTY = "26-30"
+    THIRTYONE_TOT_THIRTYFIVE = "31-35"
+    THIRTYSIX_TO_FOURTY = "36-40"
+    OVER_FOURTY = "over_40"
+
+    TIME_PER_WEEK_OPTIONS = (
+        (NOT_SPECIFIED, "Not specified"),
+        (ONE_TO_TWO, "1-2"),
+        (THREE_TO_FIVE, "3-5"),
+        (SIX_TO_TEN, "6-10"),
+        (ELEVEN_TO_FIFTEEN, "11-15"),
+        (SIXTEEN_TO_TWENTY, "16-20"),
+        (TWENTYONE_TO_TWENTYFIVE, "21-25"),
+        (TWENTYSIX_TO_THIRTY, "26-30"),
+        (THIRTYONE_TOT_THIRTYFIVE, "31-35"),
+        (THIRTYSIX_TO_FOURTY, "36-40"),
+        (OVER_FOURTY, "More than 40")
+    )
+
+    time_per_week = models.CharField(
+        help_text="Shows how many hours per week the user is putting into this specific project.",
+        verbose_name="Time per week",
+        max_length=64,
+        choices=TIME_PER_WEEK_OPTIONS,
+        default=NOT_SPECIFIED
+    )
```

Comment:
I think you should use availability table here. Its already created.

Reply:
You're right. I changed it.